kolibri/plugins/learn/assets/src/views/SuggestedTime.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+
+  <div class="time-context">
+    <div v-if="time > 0">
+      {{ $tr('shortTime', { time: 120 }) }}
+    </div>
+    <div v-if="time > 0">
+      {{ $tr('suggestedTimeLabel', { time: 120 }) }}
+    </div>
+    {{ $tr('suggestedTimeTooltip') }}
+  </div>
+
+</template>
+
+
+<script>
+
+  export default {
+    name: 'SuggestedTime',
+    props: {
+      /* Time in ms */
+      time: {
+        type: Number,
+        default: 0,
+      },
+    },
+    $trs: {
+      suggestedTimeTooltip: {
+        message: 'Suggested time to complete',
+        context:
+          'A tooltip explaning what the clock icon and time indicates about the resource  they are viewing',
+      },
+      shortTime: {
```

Comment:
@nucleogenesis Is there a way to reuse the existing `minute/minutes` string from the [TimeDuration](https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/views/TimeDuration.vue#L42) component? 

If I'm understanding correctly, you could even combine the `suggestedTimeTooltip` message with the existing `minutes` from `TimeDuration` component to achieve the purpose of `sugestedTimeLabel`, and effectively need to stub just one string here 😉